### PR TITLE
Editor: Fix workaround when exporting.

### LINKED ISSUE #6594: threejs.org/editor does not save stuff in ff (nightly)
works in chrome, but nothing happens in ff. no errors in console or anything.

### PR DESCRIPTION
**Pull request by Maxsior**
Tested on Chrome and Firefox 38 and some upper ver.

### CODE REVIEW

File: editor/js/Menubar.File.js [hunk 1]
```diff
@@ -393,14 +393,14 @@ Menubar.File = function ( editor ) {
 	//
 
 	var link = document.createElement( 'a' );
-	link.style.display = 'none';
-	document.body.appendChild( link ); // Firefox workaround, see #6594
+	link.style.display = 'none';
```
Comment (Mugen87) [inline editor/js/Menubar.File.js]: Is this line still necessary? I mean the link is not appended to the DOM anymore, right?

**Comment by Mugen87** [comment]
I've have tested your change here: http://raw.githack.com/Maxsior/three.js/patch-1/editor/index.html

It's seems it is not possible to export two times in a row. For example, the export to `JSON` works but if you want export too `glTF` after that, nothing happens.

**Comment by Maxsior** [comment]
> 
> I've have tested your change here: http://raw.githack.com/Maxsior/three.js/patch-1/editor/index.html
> 
> It's seems it is not possible to export two times in a row. For example, the export to `JSON` works but if you want export too `glTF` after that, nothing happens. Not sure working with `MouseEvent` is a good idea since it seems to work only once.

It seems MouseEvent object must be created for each click. 
Dispatching events works more effectively, this test shows that:
изображение

**Comment by Mugen87** [comment]
Okay, it works correct now with the latest change 👍

File: editor/js/Menubar.File.js [hunk 2]
```diff
@@ -393,14 +393,11 @@ Menubar.File = function ( editor ) {
...
 	var link = document.createElement( 'a' );
-	link.style.display = 'none';
-	document.body.appendChild( link ); // Firefox workaround, see #6594
-
 	function save( blob, filename ) {
 
 		link.href = URL.createObjectURL( blob );
 		link.download = filename || 'data.json';
-		link.click();
+		link.dispatchEvent( new MouseEvent('click') );
```
Comment (Mugen87) [inline editor/js/Menubar.File.js]: Can you please ensure correct code style. Write it like so:
```js
link.dispatchEvent( new MouseEvent( 'click' ) );
```

Comment (Maxsior) [reply editor/js/Menubar.File.js]: No problem. Done.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**